marketplace/optimizely/src/index.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { render } from 'react-dom';
+import { init, locations } from 'contentful-ui-extensions-sdk';
+import './index.css';
+import EditorPage from './EditorPage';
+import Sidebar from './Sidebar';
+import { IncorrectContentType, isValidContentType, MissingProjectId } from './errors-messages';
+import OptimizelyClient from './optimizely-client';
+import AppPage from './AppPage';
+import '@contentful/forma-36-react-components/dist/styles.css';
```

Comment:
You'll also need to import from Forma fcss to make sure headings/paragraphs have correct spacing. Shouldn't be needed soon but right now it's still required